doc/nrf/app_dev/config_and_build/index.rst

```diff
@@ -45,3 +45,4 @@ Make sure to consider :ref:`app_bootloaders` and :ref:`app_dfu` already at this
    multi_image
    building
    output_build_files
+   ../bootloaders_dfu/mcuboot_nsib/qspi_xip_split_image
```

Comment:
Apologies, this placement is not allowed. If this guide is about MCUboot with NSIB, this line needs to be added in `nrf/app_dev/bootloaders_dfu/index.rst`. If it has nothing to do with Bootloaders and DFU, it should be moved to `nrf/app_dev/config_and_build/`.